picard/formats/id3.py

```diff
@@ -459,6 +476,21 @@ def _save(self, filename, metadata):
                     desc = ''
                 for value in values:
                     tags.add(id3.USLT(encoding=encoding, desc=desc, text=value))
+            elif name.startswith('syncedlyrics:') or name == 'syncedlyrics':
+                if ':' in name:
+                    desc = name.split(':', 1)[1]
+                else:
+                    desc = ''
+                for value in values:
+
+                    sylt_lyrics = []
+                    timestamp_and_lyrics = batched(re.split(self.__lrc_re_parse, value)[1:], 2)
```

Comment:
This appears to assume that the metadata is in valid LRC format. If the user edits the data or if the data has been provided by a plugin this may not be the case. Also we need to handle newlines as described in an earlier comment.